packages/dx-chart-core/src/plugins/theme-manager/computeds.js

```diff
@@ -0,0 +1,10 @@
+
+const Palette = ['#2196F3', '#F44336', '#4CAF50', '#FFEB3B', '#E91E63', '#9C27B0'];
```

Comment:
I think this constant should be in theme packages because there are differences.

Reply:
Hi. We have only one palette for MUI & Bootstrap4 packages. MB we rethink it after